### PR TITLE
delete tm-monitor tool #4247

### PR DESCRIPTION
**Pull request by pushcodeveryday**
Fixes: #4247 
* [x] Referenced an issue explaining the need for the change

### CODE REVIEW

**Comment by melekes** [comment]
https://github.com/tendermint/tendermint/blob/master/docs/tools/monitoring.md
needs to be removed

docs needs to be updated

**Comment by pushcodeveryday** [comment]
> https://github.com/tendermint/tendermint/blob/master/docs/tools/monitoring.md
> needs to be removed
> 
> docs needs to be updated

Done

**Comment by marbar3778** [comment]
also: https://github.com/tendermint/tendermint/blob/master/docs/tendermint-core/running-in-production.md#monitoring-tendermint & https://github.com/tendermint/tendermint/blob/master/rpc/lib/doc.go#L85

**Comment by pushcodeveryday** [comment]
> also: https://github.com/tendermint/tendermint/blob/master/docs/tendermint-core/running-in-production.md#monitoring-tendermint & https://github.com/tendermint/tendermint/blob/master/rpc/lib/doc.go#L85
> 
> and you can remove gox from tools.mk now since tm-monitor was the only place it was used: https://github.com/tendermint/tendermint/blob/master/tools.mk#L70
> 
> thank you

done

File: tools.mk [hunk 1]
```diff
@@ -66,12 +66,6 @@ $(PROTOBUF):
 	@echo "Get GoGo Protobuf"
 	@go get github.com/gogo/protobuf/protoc-gen-gogo@v1.3.1
 
-# used to build tm-monitor binaries
```
Comment (marbar3778) [inline tools.mk]: need to remove it above as well, 
line 45 and from line 50, sorry cant comment there, github 😢

Comment (pushcodeveryday) [reply tools.mk]: done

**Comment by marbar3778** [comment]
just for heads up the tests are failing because of: panic: `listen tcp 0.0.0.0:38567: bind: address already in use`

**Pull request merged**